Mathlib/CategoryTheory/Abelian/GrothendieckAxioms.lean

```diff
@@ -92,6 +88,38 @@ lemma hasExactColimitsOfShape_of_equiv {J J' : Type*} [Category J] [Category J']
   haveI : HasColimitsOfShape J' C := hasColimitsOfShape_of_equivalence e
   ⟨preservesFiniteLimits_of_natIso (Functor.Final.colimIso e.functor)⟩
 
+variable {C} in
+lemma hasExactColimitsOfShape_obj_of_equiv (J : Type*) {D : Type*} [Category J] [Category D]
+    (e : C ≌ D) [HasColimitsOfShape J C] [HasExactColimitsOfShape J C] :
+    haveI : HasColimitsOfShape J D := Adjunction.hasColimitsOfShape_of_equivalence e.inverse
+    HasExactColimitsOfShape J D := by
+  haveI : HasColimitsOfShape J D := Adjunction.hasColimitsOfShape_of_equivalence e.inverse
+  refine ⟨⟨fun _ _ _ => ⟨@fun K => ?_⟩⟩⟩
+  refine preservesLimit_of_natIso K (?_ : e.congrRight.inverse ⋙ colim ⋙ e.functor ≅ colim)
+  apply e.symm.congrRight.fullyFaithfulFunctor.preimageIso
+  exact isoWhiskerLeft (_ ⋙ colim) e.unitIso.symm ≪≫ (preservesColimitNatIso e.inverse).symm
+
+universe v₁ u₁ v₂ u₂ in
+theorem comp_const (J : Type u') [Category.{v'} J] (C : Type u₁) [Category.{v₁} C]
```

Comment:
We already have `compConstIso`.

Reply:
Oh, I forgot to remove this auxiliary theorem now that I don't need it anymore, thanks.